src/components/search/concept-modal/concept-modal.js

```diff
@@ -98,12 +99,78 @@ export const ConceptModalBody = ({ result }) => {
   )
 
   const tabs = {
-    'overview':     { title: 'Overview',         icon: <OverviewIcon />,         content: <OverviewTab result={ result } />, },
-    'studies':      { title: studyTitle,         icon: <StudiesIcon />,          content: <StudiesTab studies={ studies } />, },
-    'cdes':         { title: cdeTitle,           icon: <CdesIcon />,             content: <CdesTab cdes={ cdes } cdeRelatedConcepts={ cdeRelatedConcepts } loading={ cdesLoading } /> },
-    'kgs':          { title: 'Knowledge Graphs', icon: <KnowledgeGraphsIcon />,  content: <KnowledgeGraphsTab graphs={ graphs } />, },
-    'explanation':  { title: 'Explanation',      icon: <ExplanationIcon />,      content: <ExplanationTab result={ result } /> },
-    'tranql':       { title: 'TranQL',           icon: <TranQLIcon />,           content: <TranQLTab result={ result } graphs = { graphs } /> }
+    'overview': {
+      title: 'Overview',
+      icon: <OverviewIcon />,
+      content: <OverviewTab result={ result } />,
+      tooltip: <div>
+        The Overview tab provides a brief description of the concept.
+        Hovering over the <InfoCircleOutlined style={{ color: "rgba(0, 0, 0, 0.45)", margin: "0 2px" }} />
+        &nbsp;besides the concept name displays the corresponding ontological
+        identifier, and where applicable, a link to the entry on the ontology&apos;s website.
+      </div>
+    },
+    'studies': {
+      title: studyTitle,
+      icon: <StudiesIcon />,
+      content: <StudiesTab studies={ studies } />,
+      tooltip: <div>
+        The Studies tab displays studies referencing or researching the concept.
```

Comment:
```suggestion
        The Studies tab displays studies whose data dictionaries reference this concept.
```

Reply:
is it important/relevant to use such a technical term here in a tooltip? I think the purpose is more to offer users a brief explanation of a tab's purpose, which this could be counterproductive to and just confuse them more.

Reply:
I agree with Griffin. This has to be in as laymen terms as possible. People don't know what a data dictionary is as it is!